docs/index.md

```diff
@@ -0,0 +1,58 @@
+# nbgitpuller
+
+`nbgitpuller` lets you distribute content to a Jupyter user via the click of a button!
+This allows your users to focus on the content without needing to understand `git`
+or other version control machinery.
+
+`nbgitpuller` provides {ref}`automatic, opinioned conflict resolution <topic/automatic-merging>`
+by using `git` under the hood.
+It is commonly used to distribute content to multiple users of a JupyterHub, though it works just fine on an individual person's computer, if they have Jupyter installed.
+
+Here's an example of `nbgitpuller` in action:
+
+```{image} _static/nbpuller.gif
+```
+
+## Use `nbgitpuller`
+
+See [](use.md) for information about how to use `nbgitpuller`.
+Here's a short overview:
+
+1. Ensure your user has `nbgitpuller` installed. This is commonly done by installing it for all users of a JupyterHub (or, you may ask them to install it manually). See [the installation instructions](install.rst) for more information.
```

Comment:
```suggestion
1. Ensure your user has `nbgitpuller` installed. This is commonly done by installing it for all users of a JupyterHub. See [the installation instructions](install.rst) for more information.
```

Manual installation doesn't actually work inside containers, as the notebook server process needs to be restarted for it to recognize the server extension.